src/main/java/org/fundacionjala/coding/german/EANValidator.java

```diff
@@ -0,0 +1,35 @@
+package org.fundacionjala.coding.german;
+
+/**
+ * Created by German on 8/29/2017.
+ */
+public class EANValidator {
+    private static final int NUM_MOD = 10;
+
+    /**
+     * @param number String
+     * @return boolean
+     */
+    public boolean validate(String number) {
+        int checkSum = sumEO(number, 0, 1) + sumEO(number, 1, 3);
+        int numericValue = Character.getNumericValue(number.charAt(12));
+
+        return numericValue == 0 && (checkSum) % NUM_MOD == 0
+                || numericValue == (NUM_MOD - (checkSum % NUM_MOD));
```

Comment:
In order to understand better the code
try this.

```java
int sum = sumEO(number, 0, 1) + sumEO(number, 1, 3);
int numericValue = Character.getNumericValue(number.charAt(12));
int checkSum = sum % NUM_MOD == 0 ? 0 : NUM_MOD - sum % NUM_MOD;
return numericValue == checkSum;
```

Reply:
keep in mind your code should be easy to understand.

Reply:
thanks, this change also helped to increase the coverage